data_collector/normalize.py

```diff
@@ -1,222 +1,66 @@
-import re
 import logging
-import numpy as np
-import pandas as pd
-from typing import Dict, List
+from typing import Dict
```

Comment:
Without having alignment on `metrics.yml` its hard to look at this new logic which isn't preserving the previous work we have. Would you mind taking small changes in incremental PRs for this purpose please?Because the logic is complex I just want us to have every step thoroughly examined.